### PR TITLE
fix: forgot to commit code

### PR DESCRIPTION
**Pull request by RaresGeo**
Looks like there was a bunch of code I accidentally didn't push for the last PR

### CODE REVIEW

File: src/tests/path.test.ts [hunk 1]
```diff
@@ -64,6 +65,14 @@ describe("Cloning", () => {
           encoding: "utf8",
         });
 
+        const mythOutput = execSync(
+          `myth analyze --execution-timeout 10 -c ${rootContractPath}`,
+          {
+            cwd: projectPath,
+            encoding: "utf8",
+          }
+        );
```
Comment (hitsuji-haneta) [inline src/tests/path.test.ts]: Why do you execute mytril here?

**Review by hitsuji-haneta — COMMENTED** [review]
@RaresGeo 
Thanks, check my comment pls :)

File: src/tests/path.test.ts (hunk 1 above)
Comment (RaresGeo) [reply src/tests/path.test.ts]: Oh, sorry. That's some leftover code from writing tests 😁

**Review by hitsuji-haneta — APPROVED** [review]
@RaresGeo 
LGTM, thank you!

**Comment by hitsuji-haneta** [comment]
oh, please update the tag for `bunzz --version` command before merging :)

**Pull request merged**